composer.json

```diff
@@ -25,24 +27,19 @@
     },
     "autoload": {
         "psr-4": {
-            "Practically\\PsalmPluginYii2\\": "src"
+            "Practically\\PsalmPluginYii2\\": "src/"
         }
     },
     "autoload-dev": {
         "psr-4": {
             "Practically\\PsalmPluginYii2\\Tests\\": [
-                "tests/_support",
-                "tests"
+                "tests/_support/",
+                "tests/"
             ]
         }
     },
-    "repositories": [
-        {
-            "type": "composer",
-            "url": "https://asset-packagist.org"
-        }
-    ],
     "scripts": {
+        "post-install-cmd": "psalm-plugin enable practically/psalm-plugin-yii2",
```

Comment:
I don't think this is necessary, this package does not use the plugin their for does not need to be enabled. We have it defined in all the [test when they run](https://github.com/Practically/PsalmPluginYii2/blob/1.x/tests/acceptance/ActiveRecordAll.feature#L21-L23).

Can you give some more context for the change?

Reply:
How your plugin can work without adding to psalm config? pass as option?

Reply:
You need to enable the plugin on a yii2 application. This will enable the plugin in **this** project's `composer.json`. Scripts for dependencies don't run. So if you `composer require practically/psalm-plugin-yii2` that script does not run.

From the composer docs

> Note: Only scripts defined in the root package's composer.json are executed. If a dependency of the root package specifies its own scripts, Composer does not execute those additional scripts.

This script would need to do in the project's `composer.json` file. However, the `psalm.xml` is committed with the plugin enabled. So I don't see the value in this.

Reply:
@AdeAttwood 
> So if you composer require practically/psalm-plugin-yii2 that script does not run.

my fail, need move it in readme

> However, the psalm.xml is committed with the plugin enabled. So I don't see the value in this.

you told about `psalm init`? when I'm `require` package, plugin not  not auto appended to `psalm.xml`